t/with_worker-events/11-clear.t

```diff
@@ -280,3 +280,86 @@ false
 target not found
 false
 target not found
+
+
+=== TEST 6: delayed_clear() would clear tgt list when we add two checkers
```

Comment:
Does this new test fail with the current `master` branch code?

Reply:
Yeah, I can make sure this test will be failed in our master branch.